internal/providers/msentraid/himmelblau.go

```diff
@@ -81,7 +83,7 @@ func ensureBrokerClientAppInitialized(tenantID string, data *deviceRegistrationD
 	brokerClientAppInitOnce.Do(func() {
 		var msalErr *C.MSAL_ERROR
 
-		authority := C.CString("https://login.microsoftonline.com/" + tenantID)
+		authority := C.CString(authorityBaseURL + "/" + tenantID)
```

Comment:
`fmt.Sprintf` is clearer in these cases IMHO.

Reply:
uh I disagree, I find `authorityBaseURL + "/" + tenantID` easier to read and more idiomatic than `fmt.Sprintf("%s/%s", authorityBaseURL, tenantID)`. I also assume that a simple string concatenation is more efficient than string interpolation.

maybe we can avoid arguing about the two approaches and instead use `url.JoinPath(authorityBaseURL, tenantID)`

Reply:
I would have probably used `url.JoinPath(authorityBaseURL, tenantID)` :)

Reply:
done